### ISSUE TITLE
Purchases: Can't fully refund plan with domain mapping bundled

### ISSUE THREAD

**Issue by gziolo** [Automattic/wp-calypso#13494]
I had two or three users this week who had plan with domain mapping bundled and wanted to cancel and refund. Unfortunately our UI doesn't allow them to do it themselves. Instead it redirects them to the customer support. I don't see any reason why they wouldn't be able to do it themselves. It makes perfect sense in case of domain registration, but it isn't a blocker in case of domain mapping.

**Comment by fabianapsimoes** [comment]
@ehti @mikeshelton1503 just wanted to get your input on this one before prioritizing it.

**Comment by ehti** [comment]
@fabianapsimoes  This has been a pain point for quite a while now. :) Lots of reports and discussion in p2MSmN-4YT-p2.

See comment # 23658 there for a summary of that thread.

If it can be fixed, it'll be really appreciated. :)

**Comment by mikeshelton1503** [comment]
This was something we had wanted to in v1 of Purchases but didn't make sense for the MVP. Now I think it's something that could be worthwhile. It reduces support load and gives users a better taste in their mouth when leaving us (or stopping paying us).

**Comment by fabianapsimoes** [comment]
@gziolo can you estimate the amount of work needed for this?

**Comment by gziolo** [comment]
I don't know how it's implemented at the moment, but I can assume it's mostly change of the copy texts in UI and email. Plus we need to force full refund if there is only mapping bundled.

**Comment by yurynix** [comment]
@fabianapsimoes explained that:
> every plan has a domain credit
> you can use it to purchase a domain, map a domain, or do nothing
> if you purchase a domain using a plan credit, when you cancel the plan you get a partial refund (plan price - domain price) and the domain is left behind
> right now, mappings have the same behavior
> but they shouldn't
> if you purchase a domain mapping using a plan credit, when you cancel the plan you should get a full refund e have both the plan and the mapping removed

**Comment by ranh** [comment]
Commenting on @yurynix's copy from #15768, but bringing the discussion back here since I think this could use a bit more attention:

> This plan includes the custom domain mapping for example.com. The mapping will be removed together with the plan, your domain example.com will stop functioning and you'll receive a full refund of $99.

Since this change could effectively break a user's site, we should be more careful about how we present it. It should be clear that:

- The domain will no longer show the user's site.
- The site will still be available at a different address.
- The domain registration is not canceled or affected in any way.

Here's a slightly more detailed description:

> This plan includes mapping for the domain example.com. Cancelling will remove all the plan's features from your site, including the domain. 
> 
> Your site will no longer be available at example.com. Instead, it will be at example.wordpress.com. 

I think we should also include this information, but perhaps not at the top of the screen:

> The domain example.com itself is not canceled. Only the connection between WordPress.com and your domain is removed. example.com is registered with (Registrar) and you can still use it with other sites.

This copy is pretty generic, it ignores the age of the domain as well as other domains on the site. 

That said, rather than more copy, I think what this screen could really use are some visual aids to explain the domain change. Even as simple as a before/after table.

@fabianapsimoes @breezyskies @mikeshelton1503

**Comment by fabianapsimoes** [comment]
> That said, rather than more copy, I think what this screen could really use are some visual aids to explain the domain change. Even as simple as a before/after table.

I'd tackle that on a separate issue, as we really just meant this to be a quick maintenance fix. cc @yurynix for the specific copy for that case.

**Comment by yurynix** [comment]
Re:
> The domain example.com itself is not canceled. Only the connection between WordPress.com and your domain is removed. example.com is registered with (Registrar) and you can still use it with other sites.
Can we rephrase it to `is registered elsewhere` since we don't (currently) know where the domain is registered.

So overall something like that:
![screen shot 2017-07-03 at 13 16 50](https://user-images.githubusercontent.com/326402/27788592-f07771be-5ff1-11e7-89a2-c129bc9eac80.png)

**Comment by fabianapsimoes** [comment]
@ehti this has now been fixed.

**Comment by ehti** [comment]
Thanks @fabianapsimoes !
